build-system/tasks/e2e/index.js

```diff
@@ -30,28 +32,46 @@ const {
   installPackages,
 } = require('../../common/utils');
 const {cyan} = require('ansi-colors');
+const {doBrowse} = require('../browse');
+const {execOrDie} = require('../../common/exec');
+const {HOST, PORT, startServer, stopServer} = require('../serve');
 const {isTravisBuild} = require('../../common/travis');
 const {reportTestStarted} = require('../report-test-status');
-const {startServer, stopServer} = require('../serve');
 const {watch} = require('gulp');
 
-const HOST = 'localhost';
-const PORT = 8000;
 const SLOW_TEST_THRESHOLD_MS = 2500;
 const TEST_RETRIES = isTravisBuild() ? 2 : 0;
 
-async function launchWebServer_() {
-  await startServer(
+const COV_DOWNLOAD_PATH = '/coverage/download';
+const COV_OUTPUT_DIR = './test/coverage-e2e';
+const COV_OUTPUT_HTML = path.resolve(COV_OUTPUT_DIR, 'lcov-report/index.html');
+
+// Set up the e2e testing environment.
```

Comment:
Style nit: Can we convert these function-level comments to JSDoc style?

Reply:
Done in local branch. `git push` doesn't seem to like using my phone hotspot for some reason, so it won't show up here until I get on a different source of internet :(